### PR TITLE
add SK capacity

### PR DESCRIPTION
**Pull request by lesspointless**
add capacity for SK, taken from 2015 annual report: https://www.sepsas.sk/

### CODE REVIEW

File: web/app/configs/capacities.json [hunk 1]
```diff
@@ -309,5 +309,17 @@
             "wind": 3,
             "solar": 263
         }
+    },
+    "SK": {
+        "capacity": {
+            "biomass": 363,
+            "nuclear": 1940,
+            "coal": 1008,
+	    "gas" : 1093,
```
Comment (corradio) [inline web/app/configs/capacities.json]: Can you align this line properly?

File: README.md [hunk 2]
```diff
@@ -98,6 +98,7 @@ Each country has a GHG mass flow that depends on neighboring countries. In order
 - Spain: [ree.es](http://www.ree.es/sites/default/files/downloadable/preliminary_report_2014.pdf)
 - Sweden [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
 - Switzerland: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
+- Slovakia: [SEPS] (https://www.sepsas.sk/Dokumenty/RocenkySed/ROCENKA_SED_2015.pdf)
```
Comment (corradio) [inline README.md]: The countries should be ordered alphabetically, can you fix that?

**Comment by brunolajoie** [comment]
Capacities looks coherents. Always under actual production over the past 7 months. Looks good for me

File: web/app/configs/capacities.json [hunk 3]
```diff
@@ -309,5 +309,17 @@
             "wind": 3,
             "solar": 263
         }
+    },
+    "SK": {
+        "capacity": {
+            "biomass": 363,
+            "nuclear": 1940,
+            "coal": 1008,
+	          "gas" : 1093,
```
Comment (corradio) [inline web/app/configs/capacities.json]: FYI, there's a space missing, but I'll fix it.

Comment (lesspointless) [inline web/app/configs/capacities.json]: thanks, vi is evil :)

**Pull request merged**